### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by karthi07**
Project setup for Tic Tac Toe.

This is for milestone 1

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+puts 'Hello world!'
```
Comment (shshamim63) [inline bin/main.rb:3]: [Mandatory]
The first line should be 
```ruby
#!/usr/bin/env ruby
```

**Review by shshamim63 — APPROVED** [review]
## Status :+1: Approved 
___
Hi @karthi07 and @evabanegacom  :wave: ,
Almost done, just add a line to finish it. Please follow the comment to do so. After that, you can move ahead. This :+1: is from me in advance.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/)

File: bin/main.rb (hunk 1 above)
Comment (karthi07) [reply bin/main.rb:3]: thanks for the suggestion :+1:  added the line and changed file permission

**Comment by karthi07** [comment]
approved milestone 1. merging into dev branch

**Pull request merged**